src/Umbraco.Core/Security/MembershipProviderBase.cs

```diff
@@ -520,7 +521,9 @@ public override string GetPassword(string username, string answer)
 
         public override string ResetPassword(string username, string answer)
         {
-            if (EnablePasswordReset == false)
+            var auth = new HttpContextWrapper(HttpContext.Current).GetUmbracoAuthTicket();

```

Comment:
this is unnecessary, there is an extension method on HttpContext too so you don't need HttpContextWrapper